tests/integration/harvest/test_ckan_load.py

```diff
@@ -207,3 +209,88 @@ def test_create_ckan_resources(self, dol_distribution_json):
         ]
         resources = create_ckan_resources(dol_distribution_json)
         assert resources == expected_resources
+
+    def test_dcatus1_1_federal_validator_success_spatial_string(
+        self,
+        interface,
+        organization_data,
+        source_data_dcatus,
+        job_data_dcatus,
+        internal_compare_data,
+    ):
+        """
+        This test is used to ensure that sources that use
+        dcatus1.1: federal as their schema and contains a "spatial"
+        attribute as a string that it passes validation.
+        """
+        # Set up for the harvest source
+        interface.add_organization(organization_data)
+        interface.add_harvest_source(source_data_dcatus)
+        harvest_job = interface.add_harvest_job(job_data_dcatus)
+
+        # Set up the harvest source
+        harvest_source = HarvestSource(harvest_job.id)
+        # Confirm the correct schema type is used in the example
+        assert harvest_source.schema_type == "dcatus1.1: federal"
+        record = internal_compare_data["records"][0]
+        # force in "spatial" attr as string
+        record["spatial"] = "United States"
+        harvest_source.validator.validate(record)
```

Comment:
i was poking around in the validator code and it has a method `is_valid` to return a boolean. that might work better here even though it doesn't reflect what we do in code.

Reply:
https://github.com/python-jsonschema/jsonschema/blob/dd1cd339bbcca809970cbbbf01136d865b82e281/jsonschema/protocols.py#L155